src/fragments/lib/graphqlapi/js/working-with-files.mdx

```diff
@@ -1187,6 +1197,14 @@ await Promise.all(
 </Block>
 </BlockSwitcher>
 
+## Working with real-time data
+
+The recommended access patterns in these docs minimize the occurrence of records that point to non-existent and/or orphaned files. This optimizes for read latency by ensuring clients *rarely* attempt to fetch a non-existent file from Storage. However, any app that deletes files can inherently cause records *on-device* to point to non-existent files.
```

Comment:
> The recommended access patterns in these docs minimize the occurrence of records that point to non-existent and/or orphaned files.

To my eyes, this sound like our patterns are trying to "minimize the occurrence of records that point to orphaned files". And, that's just nonsensical. Our readers will figure it out, but I think the phrasing is just awkward.

On top of that, while our patterns *do* attempt to remove files instead of orphaning them, they explicitly favor creating orphans over creating mirages. If you want to mention both, I might state it more like that.

```suggestion
The recommended access patterns in these docs attempt remove deleted files, but favor leaving orphans over leaving records that point to non-existent files. This optimizes for read latency by ensuring clients *rarely* attempt to fetch a non-existent file from Storage. However, any app that deletes files can inherently cause records *on-device* to point to non-existent files.
```